src/backend/controllers/verifyController.js

```diff
@@ -0,0 +1,19 @@
+const jwt = require('jsonwebtoken');
```

Comment:
Flow strict. Also header comments similar to other controllers.

Reply:
**FLOW**